e107_languages/English/lan_sitedown.php

```diff
@@ -11,5 +11,4 @@
 */
 define("PAGE_NAME", "Site temporarily closed");
 define("LAN_SITEDOWN_00", "is temporarily closed");
-define("LAN_SITEDOWN_01", "We have temporarily closed the site for some essential maintenance. This shouldn't take too long - please check back soon, apologies for the inconvenience.");
 ?>
```

Comment:
Thank you @Michaelkielstra .  All great changes. Just wondering what happened to LAN_SITEDOWN_01. Removed by mistake?

Reply:
No, a bug in my git client -- I'd meant to edit it, but for some reason it only registered removing the old line and not adding the new one.  I'll change that now.